templates/geosafe/analysis/modal/impact_card.html

```diff
@@ -102,12 +102,3 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Download Layer" %}</h4>
         </div>
     </div>
 </div>
-{% block extra_script %}
-    {% if analysis and analysis.get_task_state != "" %}
-        <script type="text/javascript">
-            $.get('{% url 'geosafe:layer-thumbnail' %}', {'layer_id': {{    analysis.impact_layer_id  }}}, function (data) {
```

Comment:
No need to do this, because we only need layer thumbnail url. We can get it directly from `analysis.impact_layer`.
This modal dialog should only be shown if the analysis succeed. So it was guaranteed that we can get the `analysis` object